packages/flutter_tools/lib/src/isolated/native_assets/native_assets.dart

```diff
@@ -89,74 +40,79 @@ class FlutterCodeAsset {
 
   final CodeAsset codeAsset;
   final Target target;
-
-  @override
-  String toString() =>
-      'FlutterCodeAsset(codeAsset: ${codeAsset.id} ${codeAsset.file}, target: $target)';
 }
 
-/// Invokes the build of all transitive Dart packages and prepares code assets
+enum SupportedAssetTypes { codeAssets, dataAssets }
```

Comment:
Can we add documentation to this enum to at least describe what the difference is between a code asset and a data asset?